### PR TITLE
Doc mk vs code change

### PR DESCRIPTION
**Pull request by markkramerus**

### CODE REVIEW

File: docs/cimpl6ToolingReference.md [hunk 1]
```diff
@@ -29,24 +28,23 @@ The CIMPL Tooling, also called SHR-CLI (Standard Health Record Command Line Inte
...
-### Outputs
-
-Depending on the configuration options selected, the SHR-CLI produces one or more of the following outputs:
+SHR-CLI produces one or all of the following outputs, depending on configuration parameters:
 
 * [FHIR Profiles, Extensions, Value Sets](#fhir-export) that form the core content of the IG,
 * A [Logical Model](#logical-model-export) corresponding to the CIMPL class definitions, expressed as FHIR StructureDefinitions,
 * [JSON Schema](#json-schema-export) for the profiles defined by the IG,
 * A [Data Dictionary](#data-dictionary-export) that lists the Must-Support data elements in the IG, as well as value sets and value set members,
 * [Model Documentation](#model-documentation-export) in the form of a [Javadoc-like](#https://docs.oracle.com/javase/7/docs/api/) browser that allows one to see the hierarchical class relationships in the logical model.
```
Comment (mlterryMitre) [inline docs/cimpl6ToolingReference.md]: broken link. Remove the '#' from '#https://docs.oracle.com/javase/7/docs/api/'

File: docs/cimpl6ToolingReference.md [hunk 2]
```diff
@@ -400,13 +437,13 @@ The content of the /out directory depends on which exporters were selected to ru
 * [json-schema](#json-schema-export) - this directory contains schemas for the (**TO DO -- need to define the JSON schema export**)
 * [modeldoc](#model-documentation-export) - this directory contains files that present the model with a look and feel similar to [Javadoc](#https://www.oreilly.com/library/view/learn-to-program/9781680500523/f_0126.html)
```
Comment (mlterryMitre) [inline docs/cimpl6ToolingReference.md]: Broken link. Remove the '#' from '#https://www.oreilly.com/library/view/learn-to-program/9781680500523/f_0126.html'

File: docs/cimpl6ToolingReference.md (hunk 1 above)
Comment (markkramerus) [reply docs/cimpl6ToolingReference.md]: fixed.

File: docs/cimpl6ToolingReference.md (hunk 2 above)
Comment (markkramerus) [reply docs/cimpl6ToolingReference.md]: fixed.

**Pull request merged**